src/ranking_watcher/usma.rs

```diff
@@ -11,13 +11,15 @@ const RANKING_URL: &str = "https://ranking.cvgo.re/";
 
 type Result<T> = result::Result<T, Box<dyn Error + Send + Sync>>;
 
-#[derive(Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub enum PositionChangeInfo {
     New,
     Diff(i32),
 }
 
-#[derive(Debug, Eq, PartialEq)]
+const NO_CHANGE: PositionChangeInfo = PositionChangeInfo::Diff(0);
```

Comment:
I would put that inside `impl PositionChangeInfo` block, so that it's not a "file-local global" const. An alternative would be to add something like `fn is_changed(&self) -> bool {self != PositionChangeInfo::Diff(0)}`.

Reply:
noice, done